### PR TITLE
Fixed small typo in typed.md

### PR DESCRIPTION
**Pull request by AlbertEmil**
Change a minor typo in the parameters table (description for `signatures`).

### CODE REVIEW

File: docs/reference/functions/typed.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ math.typed(signatures) : function
 Parameter | Type | Description
 --------- | ---- | -----------
 `name` | string | Optional name for the typed-function
-`signatures` | Object&lt;string, function&gt; | Object with one ore multiple function signatures
+`signatures` | Object&lt;string, function&gt; | Object with one or multiple function signatures
```
Comment (harrysarson) [inline docs/reference/functions/typed.md]: Would it read better to have `one or more function signatures`?

**Comment by josdejong** [comment]
Thanks @AlbertEmil. You've made the change in an auto generated markdown file. Can you make the change in the source file typed.js which is used to generate the docs? I can also do the fix for you, its just one character.

File: docs/reference/functions/typed.md (hunk 1 above)
Comment (josdejong) [reply docs/reference/functions/typed.md]: @harrysarson I'm not sure, I'm not a native speaker...

**Comment by josdejong** [comment]
No worries @AlbertEmil , thanks for taking the time to create this PR! Every improvement is welcome, also the small ones :)

**Pull request merged**